database_connector.py

```diff
@@ -0,0 +1,35 @@
+import pymysql.cursors
+
+
+class DatabaseConnector:
+    host = 'localhost'
+    user = 'root'
+    password = ''
+    db = None
+    charset = 'utf8mb4'
+    cursorclass = pymysql.cursors.DictCursor
+    
+    def __init__(self, database=None):
+        if database != None:
+            self.db = database
```

Comment:
You can have self.db = database

If database is None, then it does not hurt setting it to None again.

Reply:
fixed